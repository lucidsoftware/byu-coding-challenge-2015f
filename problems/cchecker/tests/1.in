2 12 22 32 42 52 60 70 81
1 11 25 72 80