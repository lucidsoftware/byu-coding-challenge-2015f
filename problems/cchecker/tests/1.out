79
71
34
80
72